### PR TITLE
PUD-106 - change URL pattern for routing

### PR DESCRIPTION
**Pull request by jonnywyatt**
Person profile:
```
/person/[nomsNumber]
```

View new recall:
```
/person/[nomsNumber]/recalls/[recallId]
```

### CODE REVIEW

File: server/views/partials/primaryNav.njk [hunk 1]
```diff
@@ -10,9 +10,9 @@
             attributes: { 'data-qa': 'navLinkToDo'}
         },
         {
-            text: 'Find someone',
-            href: '/find-offender',
-            active: isTodoPage != true,
+            text: 'Find a person',
+            href: '/find-person',
+            active: isFindPersonPage == true,
```
Comment (mikedunfordEE) [inline server/views/partials/primaryNav.njk:15]: Can't spot where this is set, I'm guessing WIP as nothing is failing

File: .eslintrc.json [hunk 2]
```diff
@@ -111,6 +111,7 @@
       {
         "ts-nocheck": false
       }
-    ]
+    ],
+    "@typescript-eslint/explicit-module-boundary-types": "off"
```
Comment (mikedunfordEE) [inline .eslintrc.json:115]: I still see some disabling of this in mockRequestUtils.ts, can they be removed now this is added?

**Review by mikedunfordEE — APPROVED** [review]

File: server/views/partials/primaryNav.njk (hunk 1 above)
Comment (jonnywyatt) [reply server/views/partials/primaryNav.njk:15]: That was inadvertently stashed, have restored thanks

File: .eslintrc.json (hunk 2 above)
Comment (jonnywyatt) [reply .eslintrc.json:115]: Done

Comment (jonnywyatt) [reply .eslintrc.json:115]: hmm didn't work; will return to this

**Review by mikedunfordEE — APPROVED** [review]

**Pull request merged**